src/util/metadata/side_metadata/spec_defs.rs

```diff
@@ -60,6 +60,8 @@ define_side_metadata_specs!(
     MS_ACTIVE_CHUNK = (global: true, log_num_of_bits: 3, log_bytes_in_region: LOG_BYTES_IN_CHUNK),
```

Comment:
In my branch I did a quick fix and didn't bother to coalesce `MS_ACTIVE_CHUNK` and `CHUNK_MARK` but I think we should for this PR. It's not useful to have the same metadata under different names (they basically are the same metadata).

Reply:
`MS_ACTIVE_CHUNK` only cares about chunks with malloc in them. But with this PR, we should be able to distinguish between a chunk mapped by the non-moving immix space and the malloc space (for example). I feel like `MS_ACTIVE_CHUNK` can be subsumed by `CHUNK_MARK` now.

Reply:
It might be possible. We could simply replace `MS_ACTIVE_CHUNK` in malloc mark sweep to `CHUNK_MARK` (which might be an issue if we have other spaces that use `ChunkMap`). Or we can start using `ChunkMap` in malloc mark sweep, but this may not be an easy refactoring. Either way, I will give it a try after this PR.